filters/include/pcl/filters/functor_filter.h

```diff
@@ -16,17 +16,18 @@ namespace pcl {
 template <typename PointT, typename Function>
 constexpr static bool is_functor_for_filter_v =
     pcl::is_invocable_r_v<bool,
-                          Function,
-                          const pcl::remove_cvref_t<pcl::PointCloud<PointT>>&,
-                          pcl::index_t>;
+        Function,
+        const pcl::remove_cvref_t<pcl::PointCloud<PointT>>&,
```

Comment:
Remove the `remove_cvref_t`. It's not needed since you have the type right there